### DISCUSSION TITLE
Updating stats

### DISCUSSION THREAD

**Discussion by bobl61** [Drek282/PsychoStats#80] [General]
Can I just upload the newest files to the server (replace existing), or do I need to do a full install again?

**Comment by Rosenstein** [comment]
Yes, you can just replace files without re-install.
Nothing has changed in regards to MySql.

**Reply by bobl61** [reply]
> Yes, you can just replace files without re-install. Nothing has changed in regards to MySql.

Replaced the files, now I can't login to admin panel: 
![2023-05-19_164311](https://github.com/Drek282/PsychoStats/assets/107001969/4720cc0c-ca41-4829-92a8-c8db5f15b7b7)

**Reply by Rosenstein** [reply]
Oh ffs 😁

**Comment by Rosenstein** [comment]
On a related sidenote, Ruffle needs additional manual setup after installation. Please read here: https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configure-wasm-mime-type

This will be automated in future.

**Comment by Drek282** [comment]
@bobl61 You've had some issues before where you thought you had uploaded new versions of files and you hadn't.  Please download the latest versions of both the base PS and the cstrikes module, delete all the files in your web folder except for the config.php and upload all the files again, except for the install folder and the config.php.

Do the same for your root folder, with the exception of the stats.cfg file.  Don't forget to make stats.pl and heat.pl executable.

You could also try clearing the applicable cookies.

**Comment by bobl61** [comment]
Deleted everything except config.php. Uploaded all the files again. Cleared cookies. Same error. There has too be a way to reset the admin login, no?

**Reply by Rosenstein** [reply]
You can open table 'ps_user' in PHPMyAdmin and inspect contents, but I don't recommend editing data there unless you know what you are doing.

**Comment by Rosenstein** [comment]
Re-apply the fix that you did here: 
https://github.com/Drek282/PsychoStats/issues/77#issuecomment-1523453456

**Reply by bobl61** [reply]
> Re-apply the fix that you did here: [#77 (comment)](https://github.com/Drek282/PsychoStats/issues/77#issuecomment-1523453456)

Bingo! Why didn't I think of that? :-(

**Comment by Rosenstein** [comment]
Closing  this conversation now 🙂
